Amethyst/Managers/ScreenManager.swift

```diff
@@ -56,6 +56,16 @@ final class ScreenManager<Delegate: ScreenManagerDelegate>: NSObject, Codable {
         return layouts[currentLayoutIndex]
     }
 
+    /// The index of the current layout in the layouts array
+    var currentLayoutIndexValue: Int {
+        return currentLayoutIndex
+    }
+
+    /// Returns layout info (key and name) for all layouts in this screen manager
+    var layoutsInfo: [(key: String, name: String)] {
+        return layouts.map { (key: $0.layoutKey, name: $0.layoutName) }
+    }
```

Comment:
* Could you change layoutsInfo to use a defined struct type instead of the named tuple?
* Could you include whether or not it is the selected layout in that struct instead of exposing the index separately above?